build.sh

```diff
@@ -292,10 +293,10 @@ fi
 echo ">>> Building image type(s): ${_IMAGESTOBUILD}"
 
 if [ -z "${_SKIP_REBUILD_PRESTAGE}" ]; then
-	[ -n "${CORE_PKG_TMP}" -a -d "${CORE_PKG_TMP}" ] \
-		&& rm -rf ${CORE_PKG_TMP}
-	[ -n "${CORE_PKG_PATH}" -a -d "${CORE_PKG_PATH}" ] \
-		&& rm -rf ${CORE_PKG_PATH}
+	[ -n "${CORE_PKG_TMP}" ] && [ -d "${CORE_PKG_TMP}" ] \
```

Comment:
test (aka [) has an 'and' option, -a, I don't think it's a good idea to convert one test call in 2 test calls for no other reason